pkg/parser/yamlparser.go

```diff
@@ -248,6 +248,22 @@ func (doc *YamlDocument) CouldBeOrbReference(orbReference string) (string, bool)
 	return splittedCommand[0], true
 }
 
+// Takes the name of anything that may be in an orb and returns if it is inside an orb that we can not use
+func (doc *YamlDocument) IsFromUnfetchableOrb(name string) bool {
```

Comment:
bool variables and functions that return a bool should usually be named after the _positive_ case. It's always a bit of mental gymnastics for the developer to reason about `IsFromUnfetchableOrb == true` (so it's "true" that it's "not fetchable"?) or `!IsFromUnfetchableOrb()` (so it's "not unfetchable", i.e. "fetchable"?)

What do you think of naming it `IsOrbFetchable` and inverting `true` and `false`?

----

Sorry I'm coming in late with this suggestion, feel free to ignore... 🙇

Reply:
I got to agree on this but I decided to put it this way because I thought it made it clearer in the places where it actually used: the three places it is used, we use if in the positive way because we want to skip checks if we don't have access to the pointed orb source and I found clearer it written
```typescript
if doc.IsFromUnfetchableOrb(jobName) {
  // skip
  return
}
```
than
```typescript
if !doc.IsFromFetchableOrb(jobName) {
  // skip
  return
}
```
but I don't have a strong opinion on this so I can change it if you really think it is better this other way

Reply:
Point taken. Let’s keep like this then. :+1:

Reply:
Point taken. Let’s keep like this then. :+1: